src/utils/serverRequests.js

```diff
@@ -45,3 +45,49 @@ export const removeFromLikedVideos = async(id, dispatch) => {
         console.log(err);
     }
 }
+
+export const getWatchLaterVideos = async(dispatch) => {
```

Comment:
perfect place for these types of async reqs is either in custom hooks or inside context in which you are setting this getWatchLaterVideos. Ideally your utils functions should be pure and for general purpose only like just sorting some data or performing filtering.

Reply:
Thanks for the information, I was unaware about this. I will refactor this later.